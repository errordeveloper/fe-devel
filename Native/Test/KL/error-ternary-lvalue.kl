/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

operator entry()
{
  var Integer a = 0, b = 0;
  (false? a: b) = 1;
  report(a);
  report(b);
}
