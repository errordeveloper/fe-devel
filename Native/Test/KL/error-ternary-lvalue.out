error-ternary-lvalue.kl:8:4: error: assignment: the result of a ternary operation cannot be an l-value
